lib/file.js

```diff
@@ -161,19 +162,27 @@ class SourceFile extends File {
      * stringifyLambda({name: 'f', parameters: [['p','T']], returns: 'number'})  // { (p: T): number, __parameters: { p: T } }
      * ```
      */
-    static stringifyLambda({name, parameters=[], returns='any', initialiser, isStatic=false, kind}) {
+    static stringifyLambda({name, parameters=[], returns='any', kind, initialiser, isStatic=false, callStyles={positional:true, named:true}}) {
         const parameterTypes = parameters.map(([n, t]) => `${normalise(n)}: ${t}`).join(', ')
         const parameterTypeAsObject = parameterTypes.length
             ? createObjectOf(parameterTypes)
             : empty
-        const callableSignature = `(${parameterTypes}): ${returns}`
+        const callableSignatures = []
+        if (callStyles.positional) {
+            callableSignatures.push(`\n// positional\n(${parameterTypes}): ${returns}`)
```

Comment:
I guess we don't need the leading `\n` here, or else we'd also have to add it [here](https://github.com/cap-js/cds-typer/pull/291/files#diff-387904f6b07263914bc1b0cb291a27832cc6963e01b5d44b75460771fc25ee8fR176) for `callStyles: {positions: false, named: true}`?

```suggestion
            callableSignatures.push(`// positional\n(${parameterTypes}): ${returns}`)
```

Reply:
I guess I have added it to have a line break after the `{`:
```ts
export declare const submitOrder: { 
// positional
```

Didn't know how to this better.

Reply:
Never mind, I have moved the line break to a later place.